crates/holochain/src/conductor/conductor.rs

```diff
@@ -620,38 +623,46 @@ mod dna_impls {
                                 .map(|wasm| (wasm_hash, wasm))
                         })
                         .collect::<ConductorResult<HashMap<_, _>>>()?;
-                    let wasms = holochain_state::dna_def::get_all(txn)?
```

Comment:
This call was made twice in the same function. Instead we can reuse the value from above that we assigned to `dna_defs_with_cell_id`.